packages/lesswrong/lib/collections/users/schema.ts

```diff
@@ -2630,6 +2630,19 @@ const schema: SchemaType<DbUser> = {
     group: formGroups.privacy,
     ...schemaDefaultValue(false),
   },
+  allowAdminsToUseVotes: {
+    type: Boolean,
+    optional: true,
+    nullable: true,
+    hidden: forumTypeSetting.get() !== 'EAForum',
+    canRead: [userOwns, 'admins'],
+    canUpdate: [userOwns, 'sunshineRegiment', 'admins'],
+    canCreate: ['members'],
+    label: "Allow the EA Forum Team to use my votes for content recommendations",
+    tooltip: "Allow us to use your votes in aggregate with other users' votes to inform our content recommendation choices, such as in the weekly digest. We will never look at your votes on an individual level.",
```

Comment:
> We will never look at your votes on an individual level

If this is false in e.g. the case of investigating abusive voting behavior, then probably wants a slight rewording.

Reply:
Also, we already do this for recommendations on the post page. We either need to reword this to not include that, or we need to make recommendations on the post page aware of this option.